### ISSUE TITLE
`spine-offsitenav` should only be used as an ID once

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/fin-ad-base-theme#44]
The FAIS global nav menu is output with ID `spine-offsitenav`, which is the same ID as the default Spine offsite navigation. This should be changed so that the IDs are unique.

See: https://github.com/washingtonstateuniversity/fin-ad-base-theme/blob/bbc880f5381864b77b685df3c90fb291dc7b9aae/spine/fais_global-navigation.php#L1

See: https://www.w3.org/TR/WCAG20-TECHS/F77

**Comment by quantumJLBass** [comment]
Yeah, oversight lol.  Thanks, I'll add it to our list of changes coming.
